src/mediatypes.rs

```diff
@@ -79,6 +79,6 @@ impl MediaTypes {
                 }
             }
         }
-        .map_err(|e| Error::from(e.to_string()))
+        .expect("to_mime should be always successful")
```

Comment:
I don't follow why this cannot fail. I'd prefer to leave the `Result` in place here, and let the caller make use of the `?` operator which isn't expensive.